doc/developer/translating-enums.md

```diff
@@ -0,0 +1,278 @@
+# Translating Cryptol Enum Types into SAWCore.
+
+## Finding the code
+
+The translation of Cryptol to SAWCore is in the [cryptol-saw-core package](cryptol-saw-core);
+and the core work of translation is in [this module](cryptol-saw-core/src/Verifier/SAW/Cryptol.hs).
```

Comment:
This `[cryptol-saw-core package](cryptol-saw-core)` hyperlink 404s, as GitHub interprets that link as pointing to `doc/developer/cryptol-saw-core`. (Similarly for the `[this module](...)` link.)